schedulingApp/models.py

```diff
@@ -107,8 +105,10 @@ def create_user_profile(sender, instance, created, **kwargs):
 def save_user_profile(sender, instance, **kwargs):
     instance.profile.save()
 
+
 class Course(models.Model):
     title = models.CharField(max_length=32)
+    semester = models.CharField(max_length=16)
```

Comment:
This is fine as a placeholder, but will be updated with choices with Course implementation